src/Documentation/Tests.DocGenerator/DocParsingTests.cs

```diff
@@ -13,6 +13,8 @@
 using static Microsoft.Quantum.QsCompiler.Documentation.Testing.Utils;
 using Range = Microsoft.Quantum.QsCompiler.DataTypes.Range;
 
+#pragma warning disable 618
```

Comment:
I think it would be valuable to add a brief comment on what the disabled warning is about. E.g.:
```
// A class member was marked with the Obsolete attribute is referenced.
```

Reply:
Added!